test/server/BUILD

```diff
@@ -17,6 +17,8 @@
 
 load("@graknlabs_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
 
+exports_files(["cucumber_test.sh"])
```

Comment:
Given that Cucumber is used by both Java and JS (and Python soon (TM)), then we should rename this file ... but it needs to be a distinct filename from the script in `node_modules`. How about `cucumber-js-runner.sh`?